worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/DeviceMonitor.scala

```diff
@@ -224,9 +224,9 @@ object DeviceMonitor {
     }
   }
 
+  // unit is byte
```

Comment:
B?? bytes?

Reply:
df -B1 can return space size in byte unit